CheatSheetSetup.org

```diff
@@ -161,7 +161,7 @@
 #+BEGIN_EXPORT latex
 \renewenvironment{parallel}[1][2] % one argument, whose default value is literal `2`.
  {
-  \setlength{\columnseprule}{2pt}
+  \setlength{\columnseprule}{0.5pt}
```

Comment:
This is interesting...I looked at your repo hoping to find an up to date PDF; I'm excited to see how this looks ;-)